client/dashboard/app/i18n.tsx

```diff
@@ -50,6 +73,7 @@ export function I18nProvider( { children }: PropsWithChildren ) {
 				// `realLanguage` can be different from `language` when loading language data fails
 				// and it falls back to `en`.
 				setLoadedLocale( realLanguage );
+				setLocaleInDOM( i18n, realLanguage );
```

Comment:
This function will even be called when `use-translation-chunks` is enabled. But I think it is harmless.